src/tests/tracing/eventlistener/EventListenerThreadPool.cs

```diff
@@ -50,18 +55,17 @@ static int Main()
         {
             using (RuntimeEventListener listener = new RuntimeEventListener())
             {
-                int someNumber = 0;
-                Task[] tasks = new Task[100];
-                for (int i = 0; i < tasks.Length; i++) 
+                Overlapped overlapped = new Overlapped();
+                IOCompletionCallback completionCallback = null;
+
+                unsafe
                 {
-                    tasks[i] = Task.Run(() => { someNumber += 1; });
+                    overlapped.Pack(completionCallback);
                 }
 
                 listener.TPWaitEvent.WaitOne(TimeSpan.FromMinutes(3));
 
-                if (listener.TPWorkerThreadStartCount > 0 ||
-                    listener.TPWorkerThreadStopCount > 0 ||
-                    listener.TPWorkerThreadWaitCount > 0)
+                if (listener.TPIOPack > 0)
```

Comment:
I think it would be better to modify this test to look for all of the expected events to be raised. Even previously it's only looking for one of the three events that it's tracking. Could keep track of whether each of the expected events was raised, and set the event only after all the events are raised.

Reply:
Its great to see the new event fired in `NativeAOT` but the old events shouldn't be removed. This test runs as default in `CoreCLR` CI runs but doesn't currently run in `NativeAOT` runs since the existing events are not fired in `NativeAOT` in windows. I assume the new event will work in `CoreCLR` and the test should pass on `CoreCLR` runs. 

You can enable the test in `NativeAOT` runs in Windows by excluding the previous events being validated on `NativeAOT` runs, similar to [this ](https://github.com/dotnet/runtime/blob/main/src/tests/tracing/eventpipe/providervalidation/providervalidation.cs#L38)test, and then adding the test to the TestBuildArgs [here](https://github.com/dotnet/runtime/blob/main/eng/pipelines/runtime.yml#L295). Given there are multiple projects in the test directory, something like `test tracing/eventlistener/EventListenerThreadPool.csproj` towards the end of the `testBuildArgs` should hopefully work.